app/src/main/java/fr/free/nrw/commons/contributions/ContributionsActivity.java

```diff
@@ -342,4 +354,57 @@ public static void startYourself(Context context) {
         Intent contributionsIntent = new Intent(context, ContributionsActivity.class);
         context.startActivity(contributionsIntent);
     }
+
+    private void displayFeedbackPopup() {
+
+        Date strDate = null;
```

Comment:
The "str" prefix makes me think this is a String as I read the code.  Perhaps something like `popupMessageEndDate` might be a more informative name?